### ISSUE TITLE
With authentication turned off: crash when going to Settings > Authentication 

### ISSUE THREAD

**Issue by JeremyMahieu** [Flood-UI/flood#883]
Type: Bug Report

- [x] Try to follow the update procedure described in the README and try again before opening this issue.
- [x] Please check the [F.A.Q.](https://github.com/Flood-UI/flood/wiki/F.A.Q.).
- [x] Please check the [Troubleshooting](https://github.com/Flood-UI/flood/wiki/Troubleshooting) wiki section.

## Your Environment

* Version used:
  + Latest Docker image: wonderfall/rtorrent-flood
* Environment name and version:
  + Web browser `Chrome 85`

## Summary
When navigating to the Settings > Authentication, the screen blanks and two errors are generated.

```
main.68f82a86.js:316 TypeError: this.props.users.sort is not a function
    at a.value (main.68f82a86.js:339)
    at a.value (main.68f82a86.js:339)
    at Ni (main.68f82a86.js:316)
    at Yi (main.68f82a86.js:316)
    at ms (main.68f82a86.js:316)
    at ul (main.68f82a86.js:316)
    at sl (main.68f82a86.js:316)
    at Js (main.68f82a86.js:316)
    at main.68f82a86.js:316
    at t.unstable_runWithPriority (main.68f82a86.js:324)
```

```
main.68f82a86.js:339 Uncaught (in promise) TypeError: this.props.users.sort is not a function
    at a.value (main.68f82a86.js:339)
    at a.value (main.68f82a86.js:339)
    at Ni (main.68f82a86.js:316)
    at Yi (main.68f82a86.js:316)
    at ms (main.68f82a86.js:316)
    at ul (main.68f82a86.js:316)
    at sl (main.68f82a86.js:316)
    at Js (main.68f82a86.js:316)
    at main.68f82a86.js:316
    at t.unstable_runWithPriority (main.68f82a86.js:324)
```

## Expected Behavior
Show the screen

## Current Behavior
Crash

## Steps to Reproduce

1. Disable authentication with environment variable `DISABLE_AUTH=true`
2. Go to Settings > Authentication

**Comment by jesec** [comment]
#880 has not been merged so this issue is out of scope here.

Wonderfall’s docker image now uses my fork. The issue is that several user management API endpoints are disabled when auth is disabled and as a result authentication tab won’t work. It doesn’t gracefully handle failures so you got that blank grey screen. 

This problem is mitigated in commit https://github.com/jesec/flood/commit/8d11640bdcb73a10deb9974a66876e0c5b705c59 by not displaying authentication tab when auth is disabled.

However, it still depends on build time config variables so if the Flood is not built with the option the behavior won’t change. It is fully resolved by commit https://github.com/jesec/flood/commit/4a1997013eff4db7220e65e96b986e79bc55fc72. This change is included in 3.1 release. It seems that Wonderfall’s image still uses 3.0, though. You have to bring up this issue with them.

**Comment by JeremyMahieu** [comment]
Closing since it's since resolved and does not apply to this fork.
